hibike/setup.sh

```diff
@@ -9,6 +9,17 @@ cp utils/flash /etc/bash_completion.d/flash.sh
 echo "Setup Tab-complete"
 echo "Please restart your terminals"
 
+sudo usermod -a -G dialout $(whoami)
+
+if [ $# -eq 1 ] && [ $(echo $1 | grep -c -i ".*arduino.*\.tar") != 0 ]; then
```

Comment:
this will not match .tar.xz
`".*arduino.*(\.tar|\.tar\.xz)"` will

Either change it here, or in the print statement below (by removing the .tar.xz)